helm/hades/templates/configmap.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config
+  namespace: {{.Values.namespace.name}}
+data:
+  api_url: "https://hades.k8s.aet.cit.tum.de/api"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)